volumes/volumes_test.go

```diff
@@ -21,7 +21,7 @@ func TestDefaultVolumesFromReader(test *testing.T) {
 	if err != nil {
 		test.Errorf("failed to read config")
 	}
-	if reflect.DeepEqual(actual, fakeVolumeConfig.Volumes) {
+	if !reflect.DeepEqual(actual, fakeVolumeConfig.Volumes) {
 		test.Errorf("volumes incorrect, got: %s, want: %s.", actual, fakeVolumeConfig.Volumes)
```

Comment:
`%+v` may be?
(I don't know how good the string representations for `Volume` and `VolumeMount` are TBH)